s64da_benchmark_toolkit/netdata.py

```diff
@@ -8,6 +8,25 @@
 
 LOG = logging.getLogger()
 
+def is_netdata_set_and_runnning(config):
+    netdata_url = config.get('netdata').get('url')
+    if netdata_url:
+        try:
+            response = requests.get(netdata_url)
+            status_code = response.status_code
+            if status_code != 200:
+                LOG.error(f'Netdata url response ({netdata_url}) does not return 200, but {status_code}')
+                return False
+            return True
```

Comment:
This `return True` can be removed, along with the `else` on line 27 to simplify the logic.